### PR TITLE
Typo in PR template

### PR DESCRIPTION
**Pull request by Carreau**
Not critical but create a red underline for me when I open a PR

### CODE REVIEW

**Comment by Carreau** [comment]
@goanpeca you wrote the line,  maybe "thousando" is local slang for "miles" ? 😄

**Comment by goanpeca** [comment]
Hahaha I have copied this from
Other projects and that typo has followed me ever since 🤣

Thanks for the fix :)

**Review by goanpeca — APPROVED** [review]
Thousando for the win 🏆

**Comment by brisvag** [comment]
Actually - *adjusts glasses* - I think it's supposed to be `thousand`, without an `s` :P

**Comment by goanpeca** [comment]
>Actually - adjusts glasses - I think it's supposed to be thousand, without an s :P

Actually you are correct :-p

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Description
 <!-- What does this pull request (PR) do? Why is it necessary? -->
 <!-- Tell us about your new feature, improvement, or fix! -->
-<!-- If your change includes user interface changes, please add an image, or an animation "An image is worth a thousando words!" -->
+<!-- If your change includes user interface changes, please add an image, or an animation "An image is worth a thousands words!" -->
```
Comment (goanpeca) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
<!-- If your change includes user interface changes, please add an image, or an animation "An image is worth a thousand words!" -->
```

**Review by sofroniewn — APPROVED** [review]
LGTM

**Pull request merged**